Constants without YARD @return cause errors

**Describe the bug**

In master, constants without a YARD @return cause errors.

**To Reproduce**

Make a constant without a YARD @return.

**Expected behavior**

It treats it as untyped.

**Actual behavior**

```
[ERROR] undefined method `path' for "Organization::SYNC_INTERVAL":String
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/user_interaction.rb:364:in `method_missing'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/logging.rb:78:in `generic'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/logging.rb:152:in `omit'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:132:in `block in add_constants'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:113:in `each'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:113:in `add_constants'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:506:in `add_namespace'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:521:in `block in populate'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:521:in `each'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:521:in `populate'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/generator.rb:541:in `run'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/lib/sord/parlour_plugin.rb:70:in `generate'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/bundler/gems/sord-37961370e069/exe/sord:86:in `block (2 levels) in <top (required)>'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/runner.rb:61:in `run!'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
         /Users/peterwagenet/.rvm/gems/ruby-3.0.1/gems/commander-4.6.0/lib/commander/import.rb:7:in `block in <top (required)>'
```

**Additional information**

Internally, It looks like it's expecting a special item with a `path` but we just have a string.

--- a/lib/sord/generator.rb
+++ b/lib/sord/generator.rb
@@ -129,7 +129,7 @@ module Sord
         when :rbs
           return_tags = constant.tags('return')
           returns = if return_tags.empty?
-            Logging.omit("no YARD return type given, using untyped", constant.to_s)
+            Logging.omit("no YARD return type given, using untyped", constant)
             Parlour::Types::Untyped.new
           else
             TypeConverter.yard_to_parlour(

--- a/lib/sord/logging.rb
+++ b/lib/sord/logging.rb
@@ -73,11 +73,12 @@ module Sord
     def self.generic(kind, header, msg, item)
       return unless enabled_types.include?(kind)
 
-      if item
-        puts "#{header} (#{Rainbow(item.path).bold}) #{msg}" unless silent?
+      message = if item
+        "#{header} (#{Rainbow(item.path).bold}) #{msg}"
       else
-        puts "#{header} #{msg}" unless silent?
+        "#{header} #{msg}"
       end
+      puts message unless silent?
 
       invoke_hooks(kind, msg, item)
     end

